ansible/files/hetzner_server_nftables.conf

```diff
@@ -36,17 +37,17 @@ table inet filter {
 }
 table ip nat {
   chain PREROUTING {
-    type nat hook prerouting priority dstnat; policy accept;
-    tcp dport 80 dnat ip to $KUBENODEIP:31772
-    tcp dport 443 dnat ip to $KUBENODEIP:31773
+    type nat hook prerouting priority -100;
+    iifname { $INF_WAN, virbr0 } tcp dport 80 fib daddr type local dnat to $KUBENODEIP:31772
```

Comment:
I am a bit confused here. wouldn't this redirect traffic destined to letsencrypt back to the kubenode?